### PR TITLE
refactor: routes/index.js to routes/index.ts

### PR DESCRIPTION
**Pull request by GiovanniCaiazzo01**

### CODE REVIEW

File: backend-app/routes/index.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+authRoutes(router);
+
+router.use(authController.protect);
+
+// protected routes
+userRoutes(router);
+adminRoutes(router);
+superAdminRoutes(router);
+githubRoutes(router);
+module.exports = router;
```
Comment (bellaabdelouahab) [inline backend-app/routes/index.ts:21]: change this to : export default router

**Comment by bellaabdelouahab** [comment]
thanks for your contribution  please read our guidelines on how to updated your pr on contribution.md here is the [link](https://github.com/ISIL-ESTE/Student-Workflow-Organizer/blob/d1b3d58bcf882a4cd5956dcd99d4c99240f0c169/CONTRIBUTING.md)
@GiovanniCaiazzo01

File: backend-app/routes/index.ts (hunk 1 above)
Comment (GiovanniCaiazzo01) [reply backend-app/routes/index.ts]: done! https://github.com/ISIL-ESTE/Student-Workflow-Organizer/pull/132/commits/7bb56da52435a36c847ecc29a499963b12d3a698

**Comment by bellaabdelouahab** [comment]
thank you .
you're welcome to contribute on other issues as well.
@GiovanniCaiazzo01

**Pull request merged**